src/components/SignupForm/index.js

```diff
@@ -76,7 +73,9 @@ function SignupForm() {
         <ButtonForm type="submit" isFilled>
           Sign Up
         </ButtonForm>
-        <ButtonForm type="button">Login</ButtonForm>
+        <ButtonForm type="button" isDivider>
```

Comment:
use defaultProp `type: 'button'` on `ButtonForm` component in order to dont send this prop